src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -788,6 +813,50 @@ private void discoverSifting()
         Log.getLogger().info("Finished initiating sifter config");
     }
 
+    /**
+     * Calculate the crusher modes from the config file.
+     */
+    private void discoverEnchantments()
+    {
+        for (final String string : Configurations.gameplay.enchantments)
+        {
+            final String[] split = string.split(":");
```

Comment:
Curious about this method, how does it support modded enchantments?
Would they not have to be a ResourceLocation such as `minecolonies:customEnchant`, and wouldn’t a resource location like that break here? or are enchants not proper ResourceLocations? 
The method used below `Enchantment.getEnchantmentByLocation` implies that it’s a ResourceLocation used?

Reply:
still wondering about this, and looking at the minecraft wiki enchantments are indeed resource locations, so i don't believe this code would work properly for modded enchantments

Reply:
might have to add split by "," and then allow a resource location there. Probably needed in 1.14 anyway.

Reply:
most likely yes